sla-db/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Pull base image.
```

Comment:
@mtouloup @jbonnet we have a mongodb in the SP. I think this file just describe the same use of official mongo container, nothing new added. Do we really need this?

Reply:
Just to clarify the purpose of this sla-local database. It's main purpose is to keep mainly two (2) correlations. 1) SLA template - NS (means which SLA template is appropriate for a NS) and 2) SLA of an instantiated NS - Customer. As Felipe suggested we could have a new collection in SLA repository. (inside tng-rep)

Reply:
@mtouloup is not inside tng-rep. I think we should use the same mongodb available in the infrastructure to not replicate components.

Reply:
@mtouloup what kind of data do you need to store on the SLA Mgr side? If I remember well our discussions, these are simple relations a couple of tables in a relational DB (we're using PostgreSQL) solves well. Why MongoDB?

Reply:
@jbonnet the data we need to store are the two correlations i mentioned above. 1)  SLA template - NS, 2) SLA of an instantiated NS - Customer . So, i guess we can use  PostgreSQL we already have as you said. @felipevicens What do you think?

Reply:
@mtouloup @jbonnet ok to Postgres database. We should close this PR.